public/app/features/connections/tabs/ConnectData/NoAccessModal/NoAccessModal.tsx

```diff
@@ -0,0 +1,117 @@
+import { css } from '@emotion/css';
+import React from 'react';
+
+import { GrafanaTheme2 } from '@grafana/data';
+import { useStyles2, Modal, Icon, Button } from '@grafana/ui';
+
+import { type CardGridItem } from '../CardGrid';
+
+const getStyles = (theme: GrafanaTheme2) => ({
+  modal: css`
+    width: 500px;
+  `,
+  modalContent: css`
+    overflow: visible;
+    color: ${theme.colors.text.secondary};
+
+    a {
+      color: ${theme.colors.text.link};
+    }
+  `,
+  description: css`
+    margin-bottom: ${theme.spacing.gridSize * 2}px;
+  `,
+  bottomSection: css`
+    display: flex;
+    border-top: 1px solid ${theme.colors.border.weak};
+    padding-top: ${theme.spacing.gridSize * 3}px;
+    margin-top: ${theme.spacing.gridSize * 3}px;
+  `,
+  actionsSection: css`
+    display: flex;
+    justify-content: end;
+    margin-top: ${theme.spacing.gridSize * 3}px;
+  `,
+  warningIcon: css`
+    color: ${theme.colors.warning.main};
+    padding-right: ${theme.spacing.gridSize}px;
+    margin-top: ${theme.spacing.gridSize / 4}px;
```

Comment:
spacing function also supports decimals:
```suggestion
    margin-top: ${theme.spacing(0.25)};
```

Reply:
Sorry, I have missed it, taking care of it in a separate PR: https://github.com/grafana/grafana/pull/61684